### PR TITLE
Refactor class GameOver and Added playAgain functionality

### PR DESCRIPTION
**Pull request by JessickaMoya**

### CODE REVIEW

**Review by danielaSantaCruz — APPROVED** [review]

**Review by Raymundogs — APPROVED** [review]

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameOver.java [hunk 1]
```diff
@@ -4,19 +4,20 @@
 import java.awt.*;
 import java.awt.event.*;
 import javax.swing.ImageIcon;
+import java.util.Timer;
 
 public class GameOver {
```
Comment (jpgonzales27) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/GameOver.java:10]: good job jessi maybi this line maybe this line could be removed

**Review by jpgonzales27 — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

**Review by SaulCM — APPROVED** [review]

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameOver.java [hunk 2]
```diff
@@ -26,9 +27,11 @@ public GameOver() {
     }
 
     /**
-     * Shows game over interface.
+    * Shows game over interface.
```
Comment (GustavoZHA) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/GameOver.java:30]: Add a space in beginning for alienate the comment

**Review by GustavoZHA — APPROVED** [review]

**Review by edsonarios — APPROVED** [review]

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameOver.java (hunk 1 above)
Comment (JessickaMoya) [reply src/main/java/org/fundacionjala/at13/spaceinvaders/GameOver.java]: Thank you. I just deleted that line

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameOver.java (hunk 2 above)
Comment (JessickaMoya) [reply src/main/java/org/fundacionjala/at13/spaceinvaders/GameOver.java]: Thanks. I just did it.

**Pull request merged**